src/tests/Common/scripts/nativeaottest.sh

```diff
@@ -13,3 +13,13 @@
 exename=$(basename $2 .dll)
 chmod +x $1/native/$exename
 $_DebuggerFullPath $1/native/$exename "${@:3}"
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+source $SCRIPT_DIR/../../../../eng/testing/FindWasmHostExecutable.sh "$1/native/$2"
+
+if [ -n "${WASM_HOST_EXECUTABLE}" ]; then
+  $WASM_HOST_EXECUTABLE "$WASM_BINARY_TO_EXECUTE" "${@:3}"
+else
+  chmod +x $1/native/$exename
+  $_DebuggerFullPath $1/native/$exename "${@:3}"
```

Comment:
```suggestion
SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
source $SCRIPT_DIR/../../../../eng/testing/FindWasmHostExecutable.sh "$1/native/$2"

if [ -n "${WASM_HOST_EXECUTABLE}" ]; then
  $WASM_HOST_EXECUTABLE "$WASM_BINARY_TO_EXECUTE" "${@:3}"
else
  exename=$(basename $2 .dll)
  chmod +x $1/native/$exename
  $_DebuggerFullPath $1/native/$exename "${@:3}"
```
It looks to be running `$_DebuggerFullPath $1/native/$exename "${@:3}"` twice, which can't be right...